dora/core/client/fs/src/main/java/alluxio/client/file/cache/CacheManager.java

```diff
@@ -130,6 +131,7 @@ public static CacheManager create(AlluxioConfiguration conf,
         if (isNettyDataTransmissionEnable) {
           options.setIsAsyncWriteEnabled(false);
         }
+        MultiDimensionalMetricsSystem.setCacheStorageSupplier(pageMetaStore::bytes);
```

Comment:
the reference to page meta store is captured here

Reply:
yes. and i tracked the call stack, the `PageMetaStore` will be put in a `LocalCacheFileSystem`, which is kept in the `FileSystem.Factory.FILESYSTEM_CACHE` as a singleton. so i'm not sure if it's necessary to release the reference. after all the singleton will live until the process exits.